pkg/controller/certificate-shim/sync_test.go

```diff
@@ -23,6 +23,13 @@ import (
 	"testing"
 
 	"github.com/go-logr/logr"
+	cmacme "github.com/jetstack/cert-manager/pkg/apis/acme/v1"
+	cmapi "github.com/jetstack/cert-manager/pkg/apis/certmanager/v1"
+	cmmeta "github.com/jetstack/cert-manager/pkg/apis/meta/v1"
+	"github.com/jetstack/cert-manager/pkg/controller"
+	controllerpkg "github.com/jetstack/cert-manager/pkg/controller"
+	testpkg "github.com/jetstack/cert-manager/pkg/controller/test"
+	"github.com/jetstack/cert-manager/test/unit/gen"
```

Comment:
drive-by suggestion: does this contravene https://cert-manager.io/docs/contributing/coding-conventions/#organizing-imports ?

Reply:
My bad, I did not pay attention to the imports 😇